core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/credentials/MapToAzureCredentialParametersConverter.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.cloudbreak.converter.v4.credentials;
+
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.credentials.parameters.azure.AzureCredentialParameters;
+import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
+
+@Component
+public class MapToAzureCredentialParametersConverter extends AbstractConversionServiceAwareConverter<Map<String, Object>, AzureCredentialParameters> {
```

Comment:
I think it is changeable to something like `new Json(map).getValue(AzureCredentialParameters.class)`